### PR TITLE
Issue 74

### LINKED ISSUE #74: Search Endpoint Detail Flag
We currently reference `variation_descriptor`, `therapy_descriptor`,  `disease_descriptor`, `method`, and `supported_by` by ID in the search endpoint. We should add a `detail` flag to allow users to see all of these values.

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #74

### CODE REVIEW

File: tests/unit/test_search.py [hunk 1]
```diff
@@ -501,6 +874,48 @@ def test_multiple_parameters(query_handler):
     assert len(response['matches']['propositions']) > 1
 
 
+def test_civic_detail_flag(query_handler, civic_eid2997, eid2997_proposition,
```
Comment (jsstevenson) [inline tests/unit/test_search.py:877]: It'd probably have a really ugly method signature, but would it be possible/helpful to write a helper function to perform the logic for these detail flag tests and have the CIViC and MOA test cases call it? Might make adding more sources later a bit easier

**Review by jsstevenson — APPROVED** [review]

File: tests/unit/test_search.py (hunk 1 above)
Comment (korikuzma) [reply tests/unit/test_search.py:877]: Yup, I'll add it!

Comment (korikuzma) [reply tests/unit/test_search.py:897]: Added in this commit https://github.com/cancervariants/metakb/pull/79/commits/45bff62b7513f4d67b010ae82caa2ec758661785

**Review by MayLiu27 — APPROVED** [review]
Great work!

**Pull request merged**